pkg/skaffold/kubernetes/portforward/resource_forwarder.go

```diff
@@ -126,11 +127,15 @@ func (p *ResourceForwarder) Stop() {
 
 // Port forward each resource individually in a goroutine
 func (p *ResourceForwarder) portForwardResources(ctx context.Context, resources []*latestV1.PortForwardResource) {
-	go func() {
-		for _, r := range resources {
-			p.portForwardResource(ctx, *r)
-		}
-	}()
+	var wg sync.WaitGroup
```

Comment:
this change LGTM. The code in `main` seems like a bug since the comment says `Port forward each resource individually in a goroutine` but it isn't doing that.
I took a quick look to see if port forwarder can work with concurrent requests, this block seems a bit fishy:
https://github.com/GoogleContainerTools/skaffold/blob/2d36770ca26b9c3175b1cbd165816dfa7db22a58/pkg/skaffold/kubernetes/portforward/entry_manager.go#L120-L125

It doesn't lock properly since the `Load` and `Store` are done separately

Reply:
Good point.

Reply:
Rewrote to just use a sync.Map.